### ISSUE TITLE
shutil.copystat fails with PermissionError in WSL

### ISSUE THREAD

**Issue by 0eec6a61-1187-4a6f-b326-4b69de04b736** [python/cpython#82814]
BPO | [38633](https://bugs.python.org/issue38633)
--- | :---
Nosy | @pfmoore, @tjguk, @zware, @zooba, @ben-spiller, @peter-schmidbauer

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2019-10-29.14:34:38.648>
labels = ['3.7', '3.8', 'type-bug', 'library', 'OS-windows']
title = 'shutil.copystat fails with PermissionError in WSL'
updated_at = <Date 2022-01-20.01:11:33.016>
user = 'https://github.com/peter-schmidbauer'
```

bugs.python.org fields:
```python
activity = <Date 2022-01-20.01:11:33.016>
actor = 'sam.park'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Library (Lib)', 'Windows']
creation = <Date 2019-10-29.14:34:38.648>
creator = 'pspeter'
dependencies = []
files = []
hgrepos = []
issue_num = 38633
keywords = []
message_count = 3.0
messages = ['355655', '365377', '410998']
nosy_count = 7.0
nosy_names = ['paul.moore', 'tim.golden', 'zach.ware', 'steve.dower', 'benspiller', 'pspeter', 'sam.park']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue38633'
versions = ['Python 3.6', 'Python 3.7', 'Python 3.8']
```

</p></details>

### Linked PRs
* gh-103790

**Comment by NicolasM-Forsk** [comment]
Same error here with WSL and Ubuntu 20.04 (python 3.8.10) when using shutils.move() to move a file to an NFS server (sudo mount -t drvfs servername localfolder)

**Comment by zooba** [comment]
This seems like a nice easy contribution opportunity. Given the other errors that are silently handled, also handling EACCES seems fine to me.

**Comment by ethanhs** [comment]
Looks like this was fixed in #103790 ?
